### ISSUE TITLE
I Want To Compile and Use FamiTracker-CX

### ISSUE THREAD

**Issue by ghost** [nukep/famitracker-cx#2]
I am creating a NES style game on Debian based GNU/Linux. Would you help me figure out how to compile this program?

**Comment by Prichman** [comment]
> I am creating a NES style game on Debian based GNU/Linux. Would you help me figure out how to compile this program?

The current version of the program is not complete. It crashes sometimes and can't save your work into a file so you should use windows version.

**Comment by nukep** [comment]
Yup. This project is pretty much abandoned (and incomplete), so I wouldn't recommend using it. I found it wasn't reasonable to rebase the changes from jsr's mainline version for every release.

If you really want to compile it, it's probably a matter of ensuring you have dependencies installed, running cmake (you can typically install it using your distro's package manager), then running make. These steps will depend on your distro, however. I haven't tried it in forever, so you may encounter issues with newer distros/libraries.

**Comment by TherealGenius** [comment]
@nukep 
is this program under GPL?
is the source code available to compile for Windows as well?
which libararys are required to compile the source code?

will debugg the source code to see if i can clear out the crashes
